opendatahub-io/odh-manifests
# Redeploy all core pods upon image update

**Changes proposed in this pull request**
- 
- Updated jupyterhub configuration it order to redeploy all pods upon image update. 
- Revised the rest of deployments to ensure the rest of deployments follow the taggin image policy.

**Test instructions (required)**
1. Create a new project in your openshift dev environment
2. In the KfDef, change the manifest repo to: https://github.com/lucferbux/odh-manifests/tarball/feature/lferrnan-update-jypyterhub
3. Check if the jupyterhub workload is created successfully.
4. Changed the image tag version in the deploymentconfig yaml
5. Check if the pods are redeploying with the change

**Closing issues**
- [RHODS-564](https://issues.redhat.com/browse/RHODS-564) - Add functionality to redeploy all core RHODS pods upon image update

**Additional context**
- @vpavlin  and @LaVLaS found the solution, it's my first contribution to odh, so please feel free to discuss any issue about the implementation.
